.github/workflows/TagBot.yml

```diff
@@ -12,4 +12,4 @@ jobs:
       - uses: JuliaRegistries/TagBot@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-
+          ssh: ${{ secrets.DOCUMENTER_KEY }}
```

Comment:
Was the key and corresponding secret added to the repo?

Reply:
Yes, generated a new pair and added it. I don't think we have keys at the organization level ?